src/QUESTIONS.md

```diff
@@ -101,6 +134,80 @@ Of critical importance, you should upgrade your Morello board firmware before
 installing CheriBSD.
 
 
+### How can I upgrade CheriBSD?
+
+Currently, there are no binary upgrades for CheriBSD.
+
+The only way to upgrade a CheriBSD host is to build and install CheriBSD from
+source code.
+See the
+[wiki page](https://github.com/CTSRD-CHERI/cheripedia/wiki/HOWTO:-Build-CheriBSD-natively-on-Morello)
+for instructions how to do this.
+Note that we cannot guarantee stability when upgrading CheriBSD and it is
+important to make sure you can recover your data if an upgrade fails.
+
+
+### How can I build CheriBSD from source code?
```

Comment:
If we want to document this it should be way down at the bottom as nobody in TAP should be doing this